src/pydata_sphinx_theme/assets/styles/content/_figures.scss

```diff
@@ -15,7 +21,13 @@ figure {
     color: var(--pst-color-text-muted);
     margin-left: auto;
     margin-right: auto;
+    text-align: center;
 
+    p {
+         text-align: left;
```

Comment:
this is the only line I have some hesitation about.  It looks odd (to my eye anyway) to have L-aligned caption text for an image that is C-aligned within an R-floated box. I think I'd rather default to C-aligning captions. This isn't a super-strong feeling though, no worries if the other maintainers disagree.

Reply:
C-aligned for one line text, and L-aligned for muti-lines text, as the same of LaTex.
L-aligned muti-lines is better than C-aligned,  especially when the last line is short.

Reply:
I find that jump from L-aligned to C-aligned based on multi/single lines annoying as that means we can have multiple instances of each across a single  doc site which makes this look inconsistent.

I cannot wrap my head around how the L-aligned + C-aligned (text and image) thing looks so I will have to look at the preview tomorrow (I am only checking this on mobile).

From an accessibility and readability POV L-aligned is always suggested over C-aligned so if I were to choose one of the other I'd choose L-align always (for left to right text, then R-aligned for right to left languages).

Reply:
> L-align: for left to right languages, then R-aligned for right to left languages.

Good point. But I don't really know how to realize it. Mainly I don't know how to determine the language type. 
Could you please modify it directly.

Reply:
I don't believe we have robust LR support atm so this might be for another scoped piece of work so for now let's aim for L-aligned.